fashion-example-query/fashion/app.py

```diff
@@ -67,14 +79,23 @@ def download_data(targets, download_proxy=None):
         for v in targets.values():
             if not os.path.exists(v['filename']):
                 urllib.request.urlretrieve(v['url'], v['filename'], reporthook=lambda *x: t.update(1))
-            v['data'] = load_mnist(v['filename'])
+            if v['filename'] == './workspace/labels-original':
+                v['data'] = load_labels(v['filename'])
+            else:
+                v['data'] = load_mnist(v['filename'])
 
 
 def index():
+    for j in range(num_docs):
+        d = jina_pb2.Document()
+        d.embedding.CopyFrom(array2pb(targets['index']['data'][j]))
+        label = (targets['labels']['data'][j]).item()
+        d.tags.update({'id': label})
```

Comment:
d.tags.update({'label': label}). We do not want to use `id` but label. And previously map `label_id` to `label name` as in zalandoresearch repo